skada/tests/test_base.py

```diff
@@ -0,0 +1,74 @@
+# Author: Yanis Lalou <yanis.lalou@polytechnique.edu>
+#
+# License: BSD 3-Clause
+
+from sklearn.linear_model import LogisticRegression
+from skada import SubspaceAlignmentAdapter, make_da_pipeline
+from skada.datasets import (
+    make_shifted_datasets
+)
+
+from skada.utils import extract_source_indices
+
+
+def test_base_selector_remove_masked():
+    n_samples = 10
+    X, y, sample_domain = make_shifted_datasets(
+        n_samples_source=n_samples,
+        n_samples_target=n_samples,
+        shift='concept_drift',
+        noise=0.1,
+        random_state=42,
+    )
+
+    pipe = make_da_pipeline(
+        LogisticRegression(),
+    )
+
+    selector = pipe['logisticregression']
+    print(selector.get_params())
+    X_output, y_output, routed_params = selector._remove_masked(
+        X, y, selector.get_params()
```

Comment:
Why do we need to pass parameters if `_removed_masked` is called on the instance of the selector?

Reply:
In the `.fit()` function of the selector, we modify `routed_params` depending on the input `X`. Thus we need to feed it to `_remove_masked()` since `routed_params` could potentially be different than `self.get_params()` in that case.

Thus here I just called the function `_remove_masked()` as it is called in the `fit()` function.